package.json

```diff
@@ -194,7 +193,7 @@
     "pegjs": "0.10.0",
     "proxy-from-env": "1.0.0",
     "query-string": "^6.13.2",
-    "re2": "^1.15.4",
+    "re2": "1.17.4",
```

Comment:
we don't want `^1.17.4`?

Reply:
I have the same question... but it is better to be aligned with main.